app/assets/javascripts/katello/hosts/host_and_hostgroup_edit.js

```diff
@@ -19,9 +19,21 @@ $(document).on('ContentLoad', function(){
     KT.hosts.set_media_selection_bindings();
 });
 
+KT.hosts.environmentChanged = function() {
+  // if we don't save the currently selected view it's likely
+  // it will be undefined in toggle_installation_medium due to the CV dropdown reload
+  var previous_content_view = KT.hosts.getSelectedContentView();
+
+  KT.hosts.fetchContentViews();
+  KT.hosts.toggle_installation_medium(previous_content_view);
```

Comment:
This handler allows the Synced Content (kickstart dropdown list) to be reloaded when we change the environment